### PR TITLE
turn arguments to options(switches)

### PR DESCRIPTION
**Pull request by mahdibahramih**
I turn passed bash arguments to switch.
in this change I remove this part of the code :
subnetOrIP="$1" downloadOrUpload="$2" threads="$3" tryCount="$4" config="$5" speed="$6"
and replace it with a while loop that parses the passed switch to the command
also, I added a Help function for the cfScanner.sh script.
if you think this is helpful please merge it

### CODE REVIEW

File: bash/cfScanner.sh [hunk 1]
```diff
@@ -462,18 +462,59 @@ function fncMainCFFindIP {
...
+
+
+
+usage()
+{
+  echo -e "Usage: cfScanner [ -m|--mode  SUBNET/IP ] 
+                 [ -t|--test-type  DOWN/UP ]
+		 [ -thr|--thread <int> ]
+		 [ -try|--tryCount <int> ]
+		 [ -c|--config <configgile> ]
```
Comment (MortezaBashsiz) [inline bash/cfScanner.sh:476]: configfile

File: bash/cfScanner.sh [hunk 2]
```diff
@@ -462,18 +462,59 @@ function fncMainCFFindIP {
...
+		 [ -s|--speed <int> ] 
+		 [ -f|--custom-subnet-file <custome-ip-file> (if you chose IP mode)]\n"
```
Comment (MortezaBashsiz) [inline bash/cfScanner.sh:478]: --file instead of --custom-subnet-file

File: bash/cfScanner.sh [hunk 3]
```diff
@@ -462,18 +462,59 @@ function fncMainCFFindIP {
...
+  exit 2
+}
+
+PARSED_ARGUMENTS=$(getopt -a -n  cfScanner -o m:t:thr:try:c:s:f: --long mode:,test-type:,thread:,tryCount:,config:,speed:,custom-subnet-file: -- "$@")
```
Comment (MortezaBashsiz) [inline bash/cfScanner.sh:482]: variable name
parsedArguments

File: bash/cfScanner.sh [hunk 4]
```diff
@@ -462,18 +462,59 @@ function fncMainCFFindIP {
...
+VALID_ARGUMENTS=$?
```
Comment (MortezaBashsiz) [inline bash/cfScanner.sh:483]: variable name
validArguments

**Review by MortezaBashsiz — CHANGES_REQUESTED** [review]

**Comment by mahdibahramih** [comment]
I did changes
also, I changed README.md based on this MR

**Review by MortezaBashsiz — APPROVED** [review]

**Pull request merged**